internal/server/volume/internal/v1beta1/conversion.go

```diff
@@ -1,39 +1,39 @@
 package v1beta1
 
-// Add manual conversion functions here to override automatic conversion functions
-
-import (
-	v1beta1 "github.com/kubernetes-csi/csi-proxy/client/api/volume/v1beta1"
-	internal "github.com/kubernetes-csi/csi-proxy/internal/server/volume/internal"
-)
-
-func Convert_internal_VolumeStatsResponse_To_v1beta1_VolumeStatsResponse(in *internal.VolumeStatsResponse, out *v1beta1.VolumeStatsResponse) error {
-	out.VolumeSize = in.VolumeSize
-	out.VolumeUsedSize = in.VolumeUsedSize
-	return nil
-}
-
-func Convert_v1beta1_VolumeStatsRequest_To_internal_VolumeStatsRequest(in *v1beta1.VolumeStatsRequest, out *internal.VolumeStatsRequest) error {
-	out.VolumeId = in.VolumeId
-	return nil
-}
-
-func Convert_internal_VolumeDiskNumberResponse_To_v1beta1_VolumeDiskNumberResponse(in *internal.VolumeDiskNumberResponse, out *v1beta1.VolumeDiskNumberResponse) error {
-	out.DiskNumber = in.DiskNumber
-	return nil
-}
-
-func Convert_v1beta1_VolumeDiskNumberRequest_To_internal_VolumeDiskNumberRequest(in *v1beta1.VolumeDiskNumberRequest, out *internal.VolumeDiskNumberRequest) error {
-	out.VolumeId = in.VolumeId
-	return nil
-}
-
-func Convert_internal_VolumeIDFromMountResponse_To_v1beta1_VolumeIDFromMountResponse(in *internal.VolumeIDFromMountResponse, out *v1beta1.VolumeIDFromMountResponse) error {
-	out.VolumeId = in.VolumeId
-	return nil
-}
-
-func Convert_v1beta1_VolumeIDFromMountRequest_To_internal_VolumeIDFromMountRequest(in *v1beta1.VolumeIDFromMountRequest, out *internal.VolumeIDFromMountRequest) error {
-	out.Mount = in.Mount
-	return nil
-}
+// // Add manual conversion functions here to override automatic conversion functions
```

Comment:
so all of these are not needed?

Reply:
These were removed in v1beta2, in any case each release of csi-proxy is tied to its own API version so this file isn't used anyways. i.e. if we release v1beta3 you can only connect to it using a v1beta3 client and the server API used would be the v1beta3 API

Reply:
I removed these commented lines, probably @ddebroy can help confirm if these are actually needed in v1beta3

Reply:
these lines were needed! After I understood how the conversion functions work I made more changes here and luckily was able to test most of the endpoints in the integration tests